Makefile.dep

```diff
@@ -555,3 +530,13 @@ ifneq (,$(filter random,$(USEMODULE)))
         USEMODULE += tinymt32
     endif
 endif
+
+# include package dependencies
+-include $(USEPKG:%=$(RIOTPKG)/%/Makefile.dep)
```

Comment:
*VERY!!* inconvenient!!! ;)

Seriously, this opens the way to "magic package names" where you'd just mention "USEPKG += foo" and some Makefile automagically gets the package metadata from "github.com/riot-packages/foo.git", ...